.gitignore

```diff
@@ -33,3 +33,6 @@
 
 # Compiled decode utility
 decode_binary
+
+# Development files
```

Comment:
I also would suggest setting up your vim to put swap files in an external directory rather than your working directory (I've got a `~/.vim/tmp`, for example).